src/parser/infobox.py

```diff
@@ -27,133 +28,98 @@ def infobox_item():
+            # The rounds per second scalar is equivalent to the fire rate scalar, but only the fire rate one's intuitive application is correct because the rounds per second one implies that its 
+            # (RPS + RPS from spirit) * Fire rate 
+            # when its really 
+            # RPS * Fire rate + RPS from spirit which is equal to RPS * (Fire rate + Fire rate from spirit)
+            # Therefore, the scaling should only be displayed for fire rate. There are no other spirit scalars for stats that have final multipliers where this would be replicated.
+            if key == "RoundsPerSecond" and "FireRate" in hero_data["SpiritScaling"]:
+                firerate = "FireRate"
+                value = "0% {{ss|" + str(hero_data["SpiritScaling"][firerate]) + "}}"
+                infobox_data[firerate] = value
     
-    
-    
+        
 
     # Calculate the maximum length of keys to align the values
     max_key_length = max(len(format_key(key)) for key in infobox_data.keys())
 
     # Create the Infobox template
     infobox_template = "{{Infobox hero\n"
 
-    # Pre-determine if scalars are present
-    SpiritScaling_present = "SpiritScaling" in hero_data
-    LevelScaling_present = "LevelScaling" in hero_data
-
+    # Add infobox data to template
     for key, value in infobox_data.items():
-        # Append the spirit scaling value to the base value inside {{ss|}} template call
-        if SpiritScaling_present and any(key.endswith(scalingStat) for scalingStat in hero_data["SpiritScaling"]) and key != "RoundsPerSecond": #if it has nonzero spirit scaling
-            # Search for the scaled stat that ends with the current key
-            for scalingStat in hero_data["SpiritScaling"]:
-                if key.endswith(scalingStat):
-                    if hero_data["SpiritScaling"][scalingStat] == 0.0:
-                        break
-                    value = str(value) + " {{ss|" + str(hero_data["SpiritScaling"][scalingStat]) + "}}"
-                    break
-
-        # Append the level scaling value to the base value inside {{ls|}} template call
-        elif LevelScaling_present and any(key.endswith(scalingStat) for scalingStat in hero_data["LevelScaling"]): #if it has nonzero level scaling
-            #ls template still needs to be created; infobox uses a different formatting than this currently
-            # Search for the scaled stat that ends with the current key
-            for scalingStat in hero_data["LevelScaling"]:
-                if key.endswith(scalingStat):
-                    if hero_data["LevelScaling"][scalingStat] == 0.0:
-                        break
-                    value = str(value) + " {{ls|" + str(hero_data["LevelScaling"][scalingStat]) + "}}"
-                    break
-
-        elif format_key(key) not in parameter_order:
-            continue # Skip keys not in the parameter_order
-
+        # Localize the key
+        key = getLocalizedText(key, "spanish", "attributes")
```

Comment:
change func name to snake_case

Reply:
excellent idea

Reply:
What I meant here is use `snake_case` instead of `camelCase`. ie `getLocalizedText` -> `get_localized_text`. This is to maintain consistency across the codebase. Look through all your changes for any other camel cases to change